### PR TITLE
Adding setup.py (with install functionality) and test.py (to test imports, PyTorch, and TensorBoard), closing Issue #3

### PR DESCRIPTION
**Pull request by pmberg**

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Please make sure each PR closes at least one issue, and to use branches so you can work on multiple issues concurrently. I'd suggest the workflow of "start working on issue x" in VS Code's GitHub extension, as this assigns the issue to you, creates a new branch with the issue name, and links the PR to the issue.

**Review by nikhilwoodruff — CHANGES_REQUESTED** [review]
Thanks Peter!

Could you please:

* Add .egg-info to the .gitignore for this repo
* Also remove the tensorboard logs
* Add closed issues in this PR

File: Makefile [hunk 1]
```diff
@@ -2,6 +2,8 @@ all: build
 format:
 	black . -l 79
 	linecheck . --fix
+install:
+	pip install -e .[dev]
```
Comment (nikhilwoodruff) [inline Makefile:6]: test:
    pytest tests

Comment (nikhilwoodruff) [reply Makefile:6]: Good point- actually reweight/tests

**Pull request merged**